### DISCUSSION TITLE
[Ideas] CloudberryUI need to support cutomize storage locations.

### DISCUSSION THREAD

**Discussion by RyanWei** [apache/cloudberry#227] [Ideas / Feature Requests]
### Description

When try to init CBDB clusert with cloudberryUI, the "存储路径“ can not be adjusted to cutomized path.

### Use case/motivation

In real cases, we always need to choose different storage locations, can not just put all data at path root path. Could we fix this?

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

**Comment by tuhaihe** [comment]
Great suggestion! /cc @my-ship-it
